### PR TITLE
Create release-guide.md

### PR DESCRIPTION
**Pull request by mohitsoni**
This PR documents release process for releasing packages to DC/OS Universe.

### CODE REVIEW

File: release-guide.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Release Guide
+
+## Requirements
+- JDK 8
+- Protobuf 2.5.0
+- GNU Sed 4.2.2 + (for generating new apache-casssandra package using `build-cassandra-bin.bash`)
+
+## Clone the repo (including sub-modules)
```
Comment (nickbp) [inline release-guide.md]: Can submodules comment be omitted? I thought we weren't using those anymore

File: release-guide.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+To generate a new apache-cassandra binary package:
+
+1. Execute following
+
+  ```bash
+  $ ./build-cassandra-bin.bash
+  ```
+See [build-cassandra-bin.bash](https://github.com/mesosphere/dcos-cassandra-service/blob/master/build-cassandra-bin.bash) for more details on usage.
+2. Upload the `/path/to/dcos-cassandra-service/cassandra-bin-tmp/apache-cassandra-A.B.C-bin-dcos.tar.gz` to the designated release location.
+3. Update the universe `dcos-cassandra-service/universe` with new URL if necessary.
```
Comment (nickbp) [inline release-guide.md]: Can't hurt to link to the universe file here (resources.json I assume)

File: release-guide.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+1. Make necessary chanages to `dcos-cassandra-service/universe`
+2. Prepare RELEASE NOTES
+3. Cut a tag with following format: `X.Y.Z-A.B.C`, where `X.Y.Z` is the version of service and `A.B.C` is the version of underlying Cassandra
+
+  ```
+  git tag X.Y.Z-A.B.C
+  git push origin X.Y.Z-A.B.C
+  ```
+
+4. Trigger this job: https://jenkins.mesosphere.com/service/jenkins/view/Infinity/job/cassandra/job/0-trigger-tag/ with the tag version created in step 3.
```
Comment (nickbp) [inline release-guide.md]: These could all be presented as markdown links

File: release-guide.md (hunk 1 above)
Comment (mohitsoni) [inline release-guide.md]: Done!

File: release-guide.md (hunk 3 above)
Comment (mohitsoni) [inline release-guide.md]: Done!

File: release-guide.md (hunk 2 above)
Comment (mohitsoni) [inline release-guide.md]: Done!

File: release-guide.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+1. Make necessary chanages to [dcos-cassandra-service/universe](https://github.com/mesosphere/dcos-cassandra-service/tree/master/universe)
+2. Prepare RELEASE NOTES
+3. Cut a tag with following format: `X.Y.Z-A.B.C`, where `X.Y.Z` is the version of service and `A.B.C` is the version of underlying Cassandra
+
+  ```
+  git tag X.Y.Z-A.B.C
+  git push origin X.Y.Z-A.B.C
+  ```
+
+4. Trigger the [0-trigger-tag](https://jenkins.mesosphere.com/service/jenkins/view/Infinity/job/cassandra/job/0-trigger-tag/) job with the tag version created in step 3.
```
Comment (mgummelt) [inline release-guide.md]: Why is this manual?  The process should just be:

1) tag
2) push
2) observe that the universe PR is created

**Review by nickbp — APPROVED** [review]

File: release-guide.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- GNU Sed 4.2.2 + (for generating new apache-casssandra package using `build-cassandra-bin.bash`)
+
+## Clone the repo
+```
+$ git clone git@github.com:mesosphere/dcos-cassandra-service.git
+```
+
+## Build instructions
+```
+$ ./gradlew clean build
```
Comment (mgummelt) [inline release-guide.md]: Why do I need to build locally for a release?  Shouldn't CI handle it?

File: release-guide.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+# Release Guide
```
Comment (mgummelt) [inline release-guide.md]: I think we should put this in our wiki instead of in Github.  This isn't relevant to non-Mesosphere contributers

Comment (mohitsoni) [inline release-guide.md]: I thought about it. And, figured it's better to keep everything in one-place. 

I don't mind moving this to our wiki.

File: release-guide.md (hunk 5 above)
Comment (mohitsoni) [inline release-guide.md]: The CI job will do it anyways. Let me update it.

File: release-guide.md (hunk 4 above)
Comment (mohitsoni) [inline release-guide.md]: The plumbing between `0-trigger-tag` and `release-to-universe` is missing right now. We can certainly improve it.

**Comment by gabrielhartmann** [comment]
@triclambert, @mohitsoni, @mgummelt.  If this is moving to the wiki, we should close this PR.

**Review by mgummelt — CHANGES_REQUESTED** [review]

File: release-guide.md (hunk 6 above)
Comment (mgummelt) [inline release-guide.md]: Either is fine with me

File: release-guide.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+3. Cut a tag with following format: `X.Y.Z-A.B.C`, where `X.Y.Z` is the version of service and `A.B.C` is the version of underlying Cassandra
+
+  ```
+  git tag X.Y.Z-A.B.C
+  git push origin X.Y.Z-A.B.C
+  ```
+
+4. Trigger the [0-trigger-tag](https://jenkins.mesosphere.com/service/jenkins/view/Infinity/job/cassandra/job/0-trigger-tag/) job with the tag version created in step 3.
+5. Once build job in step 4 finishes successfully, take the stub universe from the [1-build-linux](https://jenkins.mesosphere.com/service/jenkins/view/Infinity/job/cassandra/job/1-build-linux/) job 
+6. Use version from step 3 and stub universe URL from step 5 to trigger [release-to-universe](https://jenkins.mesosphere.com/service/jenkins/view/Infinity/job/infinity-tools/job/release-to-universe/) job.
```
Comment (mgummelt) [inline release-guide.md]: 7. Merge the Universe PR

**Review by mgummelt — CHANGES_REQUESTED** [review]

File: release-guide.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+# Release Guide
```
Comment (mgummelt) [inline release-guide.md]: What about releasing DSE?

**Review by mgummelt — CHANGES_REQUESTED** [review]

**Comment by NimaVaziri** [comment]
Moved this to the wiki so closing.

**Pull request closed without merging**